### PR TITLE
[DEV-3886] Refactor Prayer Menu Tile Cards

### PR DESCRIPTION
**Pull request by richarddubay**
### What does this PR do, or why is it needed?
This PR refactors the prayer menu cards to be shorter and wider than before. It also changes the colors of the selected and deselected states. All this is based off the mocks.

### How do I test this PR?
Make sure you're pointed to `alpha-rock`.
Start up the sim and verify the colors and height/width of the buttons.
Note: You may need to uninstall/reinstall the app on the sim in order to pick up the colors. I noticed sometimes that caching was an issue.

---
> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android if applicable
- [x] Set a relevant reviewer

## REVIEW

- [ ] Review updates to test coverage and snapshots
- [ ] Review code through the lens of being concise, simple, and well-documented

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

---

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Review by cdanwards — DISMISSED** [review]
LGTM! My only question might be the inclusion of the recompose

File: packages/newspringchurchapp/src/prayer/PrayerMenuCard/index.js [hunk 1]
```diff
@@ -1,43 +1,42 @@
...
+import { compose } from 'recompose';
 import PropTypes from 'prop-types';
-import Color from 'color';
-import { styled } from '@apollosproject/ui-kit';
+import { styled, withTheme } from '@apollosproject/ui-kit';
 import TileImage from '../../ui/TileImage';
 
+const enhance = compose(withTheme());
+
 const Square = styled(({ theme, ...props }) => ({
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/prayer/PrayerMenuCard/index.js]: should this be called something besides "Square"?

**Comment by richarddubay** [comment]
> LGTM! My only question might be the inclusion of the recompose

Yeah @cdanwards, if there is a way to include the theme colors without that `recompose`, I'm game.

File: packages/newspringchurchapp/src/prayer/PrayerMenuCard/index.js (hunk 1 above)
Comment (richarddubay) [reply packages/newspringchurchapp/src/prayer/PrayerMenuCard/index.js]: Haha ... yeah, probably. `Tile` okay?

Comment (redreceipt) [inline packages/newspringchurchapp/src/prayer/PrayerMenuCard/index.js]:
```suggestion
const Square = styled(({ theme, transform }) => ({
```

File: packages/newspringchurchapp/src/prayer/PrayerMenuCard/index.js [hunk 2]
```diff
@@ -1,43 +1,42 @@
...
-  marginRight: theme.sizing.baseUnit * 0.75,
+  marginRight: theme.sizing.baseUnit * 0.25,
   marginTop: theme.sizing.baseUnit * 0.5,
   marginBottom: theme.sizing.baseUnit * 0.5,
-  height: 120,
+  height: 90,
   transform: props.transform,
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/prayer/PrayerMenuCard/index.js]:
```suggestion
  transform,
```

**Review by redreceipt — DISMISSED** [review]

File: packages/newspringchurchapp/src/ui/TileImage/index.js [hunk 3]
```diff
@@ -11,7 +11,7 @@ const CardView = styled(
     overflow: 'hidden',
     width: '100%',
     height: '100%',
-    aspectRatio: 1,
+    aspectRatio: 1.75,
```
Comment (redreceipt) [inline packages/newspringchurchapp/src/ui/TileImage/index.js:14]: it's strange how this isn't used anywhere else. should this component be included in the prayer folder if not?

File: packages/newspringchurchapp/src/prayer/PrayerMenuCard/index.js (hunk 1 above)
Comment (redreceipt) [reply packages/newspringchurchapp/src/prayer/PrayerMenuCard/index.js]: `Tile` and `TileImage` should prob be a little more clear as to what they are

Comment (richarddubay) [reply packages/newspringchurchapp/src/prayer/PrayerMenuCard/index.js]: What do you have in mind? I mean, they are tiles and images for the tiles. `TileImage` is actually something we brought over from core for some customizing (that I can't remember now). I'm open to other naming.

File: packages/newspringchurchapp/src/ui/TileImage/index.js (hunk 3 above)
Comment (redreceipt) [reply packages/newspringchurchapp/src/ui/TileImage/index.js:14]: I guess a better question is are we sure this isn't used anywhere else?

Comment (richarddubay) [reply packages/newspringchurchapp/src/ui/TileImage/index.js:14]: I just looked and it's not. But ... there is one use from core and one that is in another file that actually defines _another_ `TileImage`. This is ripe for a refactor.

**Review by redreceipt — APPROVED** [review]

**Pull request merged**